diff --git a/node_modules/react-native-iap/ios/IapSerializationUtils.swift b/node_modules/react-native-iap/ios/IapSerializationUtils.swift
index 800848a..88c7dfe 100644
--- a/node_modules/react-native-iap/ios/IapSerializationUtils.swift
+++ b/node_modules/react-native-iap/ios/IapSerializationUtils.swift
@@ -206,7 +206,8 @@ func serialize(_ t: Transaction) -> [String: Any?] {
 
     #if compiler(>=5.10)
     if #available(iOS 15.2, tvOS 15.2, *) {
-        result["appTransactionID"] = t.appTransactionID
+        // Commented out due to compilation error - appTransactionID not available
+        // result["appTransactionID"] = t.appTransactionID
     }
     #endif
 
